weblate/auth/models.py

```diff
@@ -138,7 +138,7 @@ class Group(models.Model):
     )
 
     language_selection = models.IntegerField(
-        verbose_name=_('Language selection'),
```

Comment:
This is confusing - this field determines how langauge selection (which is just next field) works.